main.py

```diff
@@ -138,7 +138,7 @@ def play_game(li, game_id, control_queue, engine_factory, user_profile, config):
                 if game.should_abort_now():
                     print("    Aborting {} by lack of activity".format(game.url()))
                     li.abort(game.id)
-    except (RemoteDisconnected, ConnectionError, ProtocolError, HTTPError) as exception:
+    except (RemoteDisconnected, ChunkedEncodingError, ConnectionError, ProtocolError, HTTPError) as exception:
```

Comment:
what's this doing in this pull request?

Reply:
It fixes https://github.com/careless25/lichess-bot/issues/82